src/connection/WalletConnectV2.ts

```diff
@@ -83,10 +83,7 @@ export class UniwalletConnect extends WalletConnectV2 {
 
       // Opens deeplink to Uniswap Wallet if on iOS
       if (isIOS) {
-        const newTab = window.open(`https://uniswap.org/app/wc?uri=${encodeURIComponent(uri)}`)
-
-        // Fixes blank tab opening on mobile Chrome
-        newTab?.close()
+        window.location.href = `uniswap://wc?uri=${encodeURIComponent(uri)}`
```

Comment:
are you sure we want to change the location in the current tab? if the user is trying to connect the wallet for an interaction on this page, they probably expect to be able to return to this tab and continue the desired flow

Reply:
I'm concerned about this too.

Reply:
discussed offline and it seems like window.open is broken on ios, so we need to use this method. i asked for updates to the PR description so we can validate this is working as intended.

apparently window.location.href doesn't replace the tab contents IF the URL is a deeplink

Reply:
@cartcrom could you document this as a comment inline?